backend/app/api/auth.py

```
@@ -31,46 +33,55 @@ async def register(
                 "username": user.username,
                 "id": user.id,
                 "is_active": user.is_active,
                 "created_at": user.created_at,
-                "access_token": access_token,
-                "token_type": "bearer",
+                "updated_at": user.updated_at,
             }
         )
 
     except IntegrityError:
         raise HTTPException(status_code=status.HTTP_409_CONFLICT, detail="Email already registered")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)